modules/custom/engineering_profile_helper/modules/engineering_magazine/templates/block/magazine-navigation.html.twig

```diff
@@ -1,25 +1,35 @@
 {#
-/**
- * @file
- * Profile magazine_navigation_block
- */
+@file
+Profile magazine_navigation_block
 #}
-<div{{ attributes.addClass('flex-container') }}>
+<div{{attributes.addClass('flex-container')}}>
 
-    <div class="flex-12-of-12 magazine-landing-nav">
+	<div class="flex-12-of-12 magazine-landing-nav">
 
-        <h2 class="magazine-landing-nav__item"><a href="/magazine">Research & Ideas</a></h2>
-        <div class="magazine-landing-nav__item">Topics <span id="magazine-landing-nav__topics-toggle" aria-expanded="false" aria-label="Open the Topics menu">+</span></div>
-        <div class="magazine-landing-nav__item"><a href="/magazine/issue">Issues</a></div>
+		<h2 class="magazine-landing-nav__item">
+			<a href="/magazine" tabindex="2" aria-label="Go to main magazine page">Research & Ideas</a>
```

Comment:
Are you sure about the `tabindex`s? There are a couple in this file. Did @cjwest suggest it? It bypasses our "Skip to main content" when we use the tab key.

Reply:
Good question, Jen. You don't want to use tab indexes, they can cause problems and will definitely not pass accessibility checking. 

> tabindex values of 1+ must be avoided. These elements will receive keyboard focus before elements with no tabindex value (or tabindex="0") resulting in a navigation order that is different from the visual and/or screen reader order. Instead of using tabindex, simply adjust the page's source code order to support a logical navigation and reading order, then use CSS to adjust the visual presentation.

From https://webaim.org/techniques/keyboard/tabindex

It is best to simply adjust the HTML to render in a logical order on the page.

Reply:
Thanks.  I've removed them.